### ISSUE TITLE
Important!

### ISSUE THREAD

**Issue by Tooconfident** [AbdullahDaff/prj-rev-bwfs-dasmoto#1]
https://github.com/AbdullahDaff/prj-rev-bwfs-dasmoto/blob/master/Dasmato's%20Arts%20%26%20Craft%20Project/index.html#L5

Your css file IS NOT correctly linked to your html here. This may work on your computer, but look at the path, it's all an exact path to your computer files. Once you have uploaded a poject remotely, nothing will have access to all these files on your personal computer. You need to research how to do proper file links using [relative paths](https://www.codecademy.com/en/forum_questions/553bcfc6d3292f7e1b000760). With that in mind, the correct link will look like this:

`<link href=".\resource\css\style.css" type="text/css" rel="stylesheet">`

**Comment by AbdullahDaff** [comment]
Yeah thanks a lot for the comments, I will read all the articles suggested, and be mindful of such mistakes for future projects.
